langlib/lang.stream/src/main/ballerina/src/lang.stream/stream.bal

```diff
@@ -26,10 +29,29 @@ type PureType anydata|error;
 # + data - data to be published to the stream
 #
 # Each subscriber receives a separate clone of the data.
-public function publish(stream<PureType> strm, PureType data) = external;
+public function publish(stream<PureType> strm, PureType data) {
+    function (PureType)[] funcs = streamManager.getSubscriptionFuncs(strm);
+    int noOfFuncs = arrays:length(funcs);
+    future<any>[] ftrs = [];
+    arrays:forEach(funcs, function(function (PureType) returns () func) {
+        future<any> ftr;
+        if (data is anydata) {
+            ftr = start func(values:clone(data));
+        } else {
+            ftr = start func(data);
+        }
+        ftrs[arrays:length(ftrs)] = ftr;
```

Comment:
I think it's easy to use lang.array `push` method to append to a list

Reply:
+1

Reply:
Seems like change is not there yet.

Reply:
fixed